samples/matter/common/src/dfu/smp/dfu_over_smp.cpp

```diff
@@ -6,8 +6,9 @@
 
 #include "dfu_over_smp.h"
 
-#if !defined(CONFIG_MCUMGR_TRANSPORT_BT) || !defined(CONFIG_MCUMGR_GRP_IMG) || !defined(CONFIG_MCUMGR_GRP_OS)
-#error "DFUOverSMP requires MCUMGR module configs enabled"
+#if !defined(CONFIG_MCUMGR_TRANSPORT_BT) ||                                                                            \
+	(!defined(CONFIG_SUIT) && (!defined(CONFIG_MCUMGR_GRP_IMG) || !defined(CONFIG_MCUMGR_GRP_OS)))
+#error "DFUOverSMP requires MCUMGR module configs enabled and SUIT for the nRF54H20 platform"
```

Comment:
*MCUmgr, and shouldn't it be DFU over SMP?

Reply:
Yes, I will change it.